use-cases/pci-proxy demo.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <script src="https://pay.sandbox.datatrans.com/upp/payment/js/secure-fields-1.0.0.js"></script>
+        <style>
+            .container {
+                width: 250px;
+                height: 20px;
+                border: 1px solid;
+            }
+        </style>
+    </head>
+    <body>
+        <h3>Merchant id:</h3>
+        <input type="text" id="merchantId" />
+        <button id="init">Init</button>
+        <h3>Pci Proxy form:</h3>
+        <form>
+            <div>
+                <label for="card-number-placeholder">Card Number</label>
+                <div id="card-number-placeholder" class="container"></div>
+            </div>
+            <div>
+                <label for="cvv-placeholder">Cvv</label>
+                <div id="cvv-placeholder" class="container"></div>
+            </div>
+            <button type="button" id="submit-button">Submit</button>
+        </form>
+        <h3>Changes:</h3>
+        <div id="change"></div>
+        <h3>Result:</h3>
+        <div id="result"></div>
+        <h3>Error:</h3>
+        <div id="error"></div>
+        <script>
+            let secureFields;
+
+            document.getElementById('init').addEventListener('click', () => {
+                if (secureFields) {
+                    secureFields.destroy();
+                }
+
+                const merchantId = document.getElementById('merchantId').value;
+
+                secureFields = new SecureFields();
+                secureFields.initTokenize(
+                    merchantId,
+                    {
+                        cardNumber: 'card-number-placeholder',
+                        cvv: 'cvv-placeholder',
+                    },
+                    {
+                        debug: true,
+                    }
+                );
+
+
+                secureFields.on('change', writeJson('change'));
+                secureFields.on('success', writeJson('result'));
+                secureFields.on('error', writeJson('error'));
+            })
+
+            const button = document.getElementById('submit-button');
+            button.addEventListener('click', () => {
+                if (secureFields) {
+                    secureFields.submit();
+                }
+            })
+
+            const writeJson = id => data => document.getElementById(id).innerText = JSON.stringify(data, null, 2);
+        </script>
+    </body>
+</html>
```

Comment:
This is ugly and probably wouldn't work in either in GitBook/GitHub. Ask FE guys to beutify it a little bit and also figure out where we could host it.

Reply:
I like the idea, but the current form doesn't align with us aspiring to be an unicorn company 😀

Reply:
discussed with Alex, we'll put it on codesandbox or codepen or something like that for partners to see. Will do it as a v2 of the use case